docs/tutorials/clojure-guide.md

```diff
@@ -0,0 +1,174 @@
+---
+author: ericdallo
+disqus: emacs-lsp
+---
+
+# Configuring `Emacs` as a `Clojure` IDE
+
+In this guide, I will show you how to configure emacs to develop for Clojure
+using both `lsp-mode`, which have a lot of IDE features like **completion**, **renaming**
+**find-references**, **find-definition**, **code lens**, **code actions**... and [cider](https://github.com/clojure-emacs/cider),
+the Emacs package most famous for a Clojure **REPL driven development**. 
+
+I thought about writing a guide on how to configure Emacs with `lsp-mode` for Clojure, but [this guide](https://emacs-lsp.github.io/lsp-mode/tutorials/CPP-guide/)
```

Comment:
can you still include minimal configuration similar to the one in the CPP guide including lsp-mode and cider?

Reply:
Done